crates/esplora/src/async_ext.rs

```diff
@@ -5,6 +5,7 @@ use bdk_core::{
     bitcoin::{BlockHash, OutPoint, ScriptBuf, Txid},
     BlockId, CheckPoint, ConfirmationBlockTime, Indexed, TxUpdate,
 };
+use esplora_client::Sleeper;
```

Comment:
@ValuedMammal Don't we need to re-export this trait, so downstream applications can re-implement it, such as `bdk-wasm`?

Reply:
We re-export the `esplora-client` crate so it can be used like this right?
```rust
use bdk_esplora::esplora_client::Sleeper;
```